.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,6 +1,7 @@
 <!--
 We appreciate the effort for this pull request but before that please make sure you read the contribution guidelines given above, then fill out the blanks below.
 
+**All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.**
```

Comment:
I thought this was a check-box in the other repos.

Reply:
It is both.